core/mapreduce/src/main/java/mil/nga/giat/geowave/mapreduce/input/GeoWaveInputFormat.java

```diff
@@ -136,7 +136,7 @@ public static DataStatisticsStore getJobContextDataStatisticsStore(
 				context);
 	}
 
-	public static void addIndex(
+	public static void setIndex(
```

Comment:
Well, I looked a little deeper, it looks while we append an index every time you call setIndex, we just seemingly arbitrarily use the first index to form query options.  Some of this was from before this pull request, but maybe something we should clean up?